core-services/prow/02_config/openshift/origin/_pluginconfig.yaml

```diff
@@ -12,6 +12,7 @@ label:
       - sttts
       - s-urbaniak
       - tkashem
+      - dtantsur
```

Comment:
nit: ideally keep the list sorted

Reply:
ok, good point, it seems that only "backport-risk-assessed" is sorted and not the "cherry-pick-approved", but let me fix it at least for the "backport-risk-assessed" one to keep it in order.

Reply:
done